frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/form/form-field-row.tsx

```diff
@@ -8,50 +8,88 @@ import { FormRow } from '../../form/form-row';
 import { FormFieldContext } from './form-field-context';
 import { FormFieldType } from './form-field';
 import { FormFieldReviewContext } from './form-field-review-context';
-import { FormFieldReviewRow } from './form-field-review-row';
+import { FormFieldReviewMemoRow, FormFieldReviewRow } from './form-field-review-row';
 
 const isLoading = (loadingResources?: { [k: string]: any }) =>
   loadingResources &&
   _.some(Object.keys(loadingResources), (key) => !iGetIsLoaded(loadingResources[key]));
 
+export const FormFieldInnerRow: React.FC<FieldFormInnerRowProps> = React.memo(
+  ({ field, fieldType, children, loadingResources, validation }) => {
+    const fieldKey = iGet(field, 'key');
+    const loading = isLoading(loadingResources);
+
+    return (
+      <FormRow
+        key={fieldKey}
+        fieldId={getFieldId(fieldKey)}
+        title={fieldType === FormFieldType.INLINE_CHECKBOX ? undefined : getFieldTitle(fieldKey)}
+        help={getFieldHelp(fieldKey, iGetFieldValue(field))}
+        isRequired={isFieldRequired(field)}
+        isHidden={isFieldHidden(field)}
+        validationMessage={validation ? undefined : iGetIn(field, ['validation', 'message'])}
+        validationType={validation ? undefined : iGetIn(field, ['validation', 'type'])}
+        isLoading={loading}
+        validation={validation}
+      >
+        <FormFieldContext.Provider value={{ field, fieldType, isLoading: loading }}>
+          {children}
+        </FormFieldContext.Provider>
+      </FormRow>
+    );
+  },
+);
+
+type FieldFormInnerRowProps = {
+  field: any;
+  fieldType: FormFieldType;
+  children?: React.ReactNode;
+  loadingResources?: { [k: string]: any };
+  validation?: ValidationObject;
+};
+
+export const FormFieldInnerMemoRow = React.memo(
+  FormFieldInnerRow,
```

Comment:
I am wondering what kind of issue this "double-memoization" is trying to solve. If the `FormFieldInnerRow` is not a pure function, wouldn't this cause unexpected skips in rendering?